src/modules/scraper/module.ts

```diff
@@ -41,6 +44,7 @@ import { DepositAcxPriceConsumer } from "./adapter/messaging/DepositAcxPriceCons
     TokenPriceConsumer,
     DepositFilledDateConsumer,
     DepositAcxPriceConsumer,
+    SuggestedFeesConsumer,
```

Comment:
Was this added twice in the providers list?

Reply:
No, one is the service `SuggestedFeesService` and the other one the consumer `SuggestedFeesConsumer`

Reply:
👍